### PR TITLE
Filter and bug fix GOG extraction

### LINKED ISSUE #171: More filtering on GOG
## Description
Filter out games which have '''Buying this game on GOG.COM you will receive a censored version of the game.'''  or '''This Game may contain content not appropriate for all ages or may not be appropriate for viewing at work.''' 

## Required Files
- extract_gog.py

## User Story
As a user I do not want to see any inappropriate games.

### PR DESCRIPTION
**Pull request by annalisev**
I have added in extra filtering in the GOG extraction script to try and remove inappropriate games.

I have also fixed some bugs that occurred while doing the above.

Closes #171 .

### CODE REVIEW

**Comment by annalisev** [comment]
I have updated tests to reflect the new code!

File: pipeline/extract/gog/extract_gog.py [hunk 1]
```diff
@@ -12,6 +12,8 @@
 
 
 PLATFORM_TITLE = 'window.productcardData.cardProductSystemRequirements'
+FILTER_1 = 'This Game may contain content not appropriate for all ages or may not be appropriate for viewing at work'
```
Comment (hhm970) [inline pipeline/extract/gog/extract_gog.py:15]: Good use of global variables as your filters; makes it easy to change if GOG ever do so in the future

File: pipeline/extract/gog/test_extract_gog.py [hunk 2]
```diff
@@ -45,10 +45,9 @@ def test_get_rating_bad_input():
 
 # get_release_date
```
Comment (hhm970) [inline pipeline/extract/gog/test_extract_gog.py:46]: Make sure commented code gets deleted before merging

File: pipeline/extract/gog/extract_gog.py [hunk 3]
```diff
@@ -163,7 +172,7 @@ def search_pages_last_day() -> list:
     load_dotenv()
     counter = 1
     games_from_page = req.get(
-        f'{ENV["GOG_BASE_URL"]}&page={counter}', timeout=5)
+        f'{ENV["GOG_BASE_URL"]}&page={counter}', timeout=1)
```
Comment (hhm970) [inline pipeline/extract/gog/extract_gog.py:175]: Just curious as to why the timeout is lower?

**Review by hhm970 — COMMENTED** [review]
I can't find very much to fault in your extract script - you've included all details, including docstrings and type hints in your functions; well done!

File: pipeline/extract/gog/extract_gog.py (hunk 3 above)
Comment (annalisev) [reply pipeline/extract/gog/extract_gog.py:175]: Good spot, this was a mistake!

**Review by hhm970 — APPROVED** [review]
I am happy with your code!

**Pull request merged**